library/src/main/java/com/okta/oidc/clients/AuthClientImpl.java

```diff
@@ -33,23 +33,26 @@
 import com.okta.oidc.util.AuthorizationException;
 
 import java.util.concurrent.Executor;
+import java.util.concurrent.Future;
 
 class AuthClientImpl implements AuthClient {
     private RequestDispatcher mDispatcher;
-    private SyncAuthClient mSyncNativeAuthClient;
+    private SyncAuthClientImpl mSyncNativeAuthClient;
```

Comment:
Why do we want replace interface with implementation ?

Reply:
not sure. probably a mistake in find and replace or auto fill